CODEOWNERS

```diff
@@ -1 +1 @@
-*       @awikkerink
+*       @dlockhart
```

Comment:
Hopefully this is OK @awikkerink! Figured you'd be happy to be off these reviews haha.

Reply:
Yes, this is fine. I haven't been working on d2l-table for a while, and I'll be happy to let it go. It's been a pain from the beginning (curse you IE11 and IE10).

Reply:
Haha yes IE11 going away definitely allowed a lot of the code in here to simply be deleted.